### PR TITLE
#165264041 User View Profile Endpoint Functionality

### PR DESCRIPTION
**Pull request by toluola**
#### Description
- A user will be able to view their Profile and others profile 

#### Type of change
-  New feature (a non-breaking change which adds functionality)

#### How Has This Been Tested?
- Integration

#### Checklist:
- [x] my code follows style guidelines for this project

#### PT
- [#165264041](https://www.pivotaltracker.com/story/show/165264041)

#### Postman Screenshot
  Screenshot 2019-04-13 at 8 43 30 PM

### CODE REVIEW

File: src/controllers/authentication/user.js [hunk 1]
```diff
@@ -58,3 +52,37 @@ export const createUser = async (req, res) => {
...
+      return res.status(200).json(responseFormat({
+        status: 'success',
+        data: {
+          user,
+        },
+      }));
+    }
+
+    return res.status(404).json(errorResponseFormat({
+      message: 'This profile does not exist',
```
Comment (Simplemart17) [inline src/controllers/authentication/user.js:87]: You might want to return `status` here.

File: src/controllers/authentication/user.js [hunk 2]
```diff
@@ -58,3 +52,37 @@ export const createUser = async (req, res) => {
...
+    }));
+  } catch (error) {
+    return res.status(500).json(errorResponseFormat({
+      message: 'Something Went Wrong',
```
Comment (Simplemart17) [inline src/controllers/authentication/user.js:100]: You might want to return `status` here.

**Review by Simplemart17 — COMMENTED** [review]
You might want to also add more test cases to increase coverage. Thanks.

File: src/controllers/authentication/user.js (hunk 1 above)
Comment (toluola) [reply src/controllers/authentication/user.js:87]: the errorResponseFormat() has a default status of error, i only need to pass message into the function

File: src/controllers/authentication/user.js (hunk 2 above)
Comment (toluola) [reply src/controllers/authentication/user.js:100]: the errorResponseFormat() has a default status of error, i only need to pass message into the function

**Review by Temmyogunbo — APPROVED** [review]

**Pull request merged**